src/main/scala/cromwell/binding/WdlImplicits.scala

```diff
@@ -0,0 +1,9 @@
+package cromwell.binding
+
+import cromwell.binding.types.{WdlIntegerType, WdlStringType}
+
+object WdlImplicits {
+
+  implicit def stringToWdlValue(string: String): WdlValue = WdlValue(string, WdlStringType)
```

Comment:
Is [this](https://github.com/broadinstitute/cromwell/commit/57672d9bf67ad5bf2ea92f9d1ea81dfc4c179b36) what you meant?  Not sure I love it... :smile: